Storage/api.h

```diff
@@ -287,8 +294,11 @@ class RESTfulStorage {
                            const std::string& restful_url_prefix,
                            impl::STORAGE_HANDLERS_MAP& handlers) {
       ForEachFieldByIndex<BLAH, I - 1>::RegisterIt(storage, restful_url_prefix, handlers);
-      handlers.insert(
+      using T_SPECIFIC_FIELD = impl::RESTfulHandlerGenerator<T_REST_IMPL, I - 1, T_STORAGE_IMPL>::T_SPECIFIC_FIELD;
+      ConditionalOn<ExcludeTypeFromPersistence<T_SPECIFIC_FIELD>::excluded>::Call([&] {
```

Comment:
`ExcludeTypeFromPersistence` in fact excludes field from RESTful interface, not persistence, right?